app/lib/registries/full_topic_taxonomy_registry.rb

```diff
@@ -21,13 +21,13 @@ def cacheable_data
     end
 
     def report_error
-      GovukStatsd.increment("#{NAMESPACE}.full_topic_taxonomy_api_errors")
+      GovukStatsd.increment("registries.full_topic_taxonomy_api_errors")
```

Comment:
Should this have changed?

Reply:
Yeah. It's an undefined constant if you run the spec for this file on its own. I meant to remove this for all registries, but missed this one.